doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
```

Comment:
What's the intended audience for this doc? The guidance below does not say much about *where* to maintain the bbappends/bbs. Depending on the audience the answer will vary.